uui/components/filters/FilterPanelItemToggler.tsx

```diff
@@ -29,6 +27,13 @@ export const FilterPanelItemToggler = React.forwardRef<HTMLDivElement, FilterToo
 
     const getTitle = props.predicateName ? `${props.title} ${props.predicateName}` : `${props.title}${props.selection ? ':' : ''}`;
 
+    const getSelectionText = () => props.selection.map((i, index) => (
+        <>
+            <Text key={ index } color="brand" size={ props.size } cx={ css.selection }>{i}</Text>
```

Comment:
You should use key prop in wrapper element for list Item (use some div instead Fragment)
Also it is a bad practice to use index as key

Reply:
or just use <Fragment key={...}

Reply:
Thank you @9vit917  for your message. I corrected it there:
https://github.com/epam/UUI/commit/5fd1e18da4d6f11a64b0cad0fc74c6da862ac3e3